pkg/crypto/rings/client.go

```diff
@@ -217,6 +219,33 @@ func (rc *ringClient) addressesToPubKeys(
 	return pubKeys, nil
 }
 
+// getRingPointsForAddressAtHeight returns a map of the ring points for the given
+// application
+func (rc *ringClient) getRingPointsForAddressAtHeight(
+	ctx context.Context,
+	appAddress string,
+	blockHeight int64,
+) (map[string]ringtypes.Point, error) {
+	ringPubKeys, err := rc.getRingPubKeysForAddress(ctx, appAddress, blockHeight)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get the points on the secp256k1 curve for the public keys in the ring.
+	points, err := pointsFromPublicKeys(ringPubKeys...)
+	if err != nil {
+		return nil, err
+	}
+
+	ringPoints := make(map[string]ringtypes.Point, len(points))
+	for _, point := range points {
+		ringPoints[string(point.Encode())] = point
```

Comment:
It's not super clear to me what `string(point.Encode())`. Is it like a hash or something else?

Trying to think if there's a better name to make this clear.